packages/react/tests/e2e-storybook/cypress/integration/Masthead/Masthead.e2e.js

```diff
@@ -292,27 +292,12 @@ describe('Masthead | custom (desktop)', () => {
   });
 
   it('should scroll the L0 overflow properly', () => {
-    cy.document().then($document => {
-      $document.querySelector('head').insertAdjacentHTML(
-        'beforeend',
-        `
-      <style>
-        /* Disable CSS transitions. */
-        * { -webkit-transition: none !important; -moz-transition: none !important; -o-transition: none !important; transition: none !important; }
-        /* Disable CSS animations. */
-        * { -webkit-animation: none !important; -moz-animation: none !important; -o-animation: none !important; animation: none !important; }
-        /* Reset values on non-opaque/offscreen framer-motion components. */
-        *[style*="opacity"] { opacity: 1 !important; }
-        *[style*="transform"] { transform: none !important; }
-      </style>
-    `
-      );
-    });
-
     cy.get('.bx--header__nav-caret-right').click();
 
-    cy.get('.bx--header__nav-caret-right-container').then($button => {
-      expect($button).to.have.attr('hidden');
+    cy.get('.bx--header__nav-caret-right-container').then(() => {
```

Comment:
Realize that this isn't checking for the hidden attribute before continuing. Will try to fix that today.